Justfile

```diff
@@ -181,7 +181,7 @@ test-spec-no-spark *ARGS: devenv
 # Run the unit tests only. Optional args are passed to pytest.
 test-unit *ARGS: devenv
     $BIN/python -m pytest tests/unit {{ ARGS }}
-    $BIN/python -m doctest tests/lib/in_memory/database.py
+    $BIN/python -m pytest --doctest-modules databuilder
```

Comment:
comment: This seems like a configuration improvement anyway, as well as a workaround for the import issue.